.drone.yml

```diff
@@ -63,7 +63,7 @@ pipeline:
     <<: *helm_deploy
     prefix: STG
     secrets: [ STG_API_SERVER, STG_KUBERNETES_TOKEN ]
-    values: gitbasePlayground.image.tag=commit-${DRONE_COMMIT_SHA:0:7},ingress.globalStaticIpName=gitbase-playground-staging,ingress.hostname=gitbase-playground-staging.srcd.run
+    values: gitbaseWeb.image.tag=commit-${DRONE_COMMIT_SHA:0:7},ingress.globalStaticIpName=gitbase-playground-staging,ingress.hostname=gitbase-playground-staging.srcd.run
```

Comment:
we didn't update helm chart yet, right?
Maybe don't change it as long as helm chart expects `gitbasePlayground`?

Reply:
yes, this PR depends on https://github.com/src-d/charts/pull/65 being merged. Actually the only thing this PR does is use that chart rename, we don't need to merge anything until then.